optimizely/optimizely.py

```diff
@@ -1159,3 +1198,31 @@ def _decide_for_keys(self, user_context, keys, decide_options=None):
                 continue
             decisions[key] = decision
         return decisions
+
+    def get_flag_variation_by_key(self, flag_key, variation_key):
+        """
+        Gets variation by key.
+        variation_key can be a string or in case of forced decisions, it can be an object.
+
+        Args:
+            flag_key: flag key
+            variation_key: variation key
+
+        Returns:
+            Variation as a map.
+        """
+        config = self.config_manager.get_config()
```

Comment:
aren't we supposed to check if config is valid or not?

Reply:
do check in the next couple of lines

Reply:
Done